samples/flow/src/Home.svelte

```diff
@@ -1,4 +1,6 @@
 <script>
+  import * as Glean from "../../../dist/glean.js";
```

Comment:
Just `import "../../../dist/glean.js"` should work here.

Reply:
Actually, I thin it would be best to move this import to the `metrics.js` file in this case. Otherwise, everytime we import Glean we will create a new instance of it.

Reply:
Yes, that worked, but it complained when loading metrics.js that it could not find "Glean"